PT/asciidoc/src/05_building_block_view.adoc

```diff
@@ -0,0 +1,225 @@
+ifndef::imagesdir[:imagesdir: ../images]
+
+[[section-building-block-view]]
+
+
+== Visão de Blocos de Construção
+
+ifdef::arc42help[]
+[role="arc42help"]
+****
+.Conteúdo
+A visão de blocos de construção mostra a decomposição estática do sistema em blocos de construção (módulos, componentes, subsistemas, classes, interfaces, pacotes, bibliotecas, frameworks, camadas, partições, níveis, funções, macros, operações, estruturas de dados, ...) bem como suas dependências (relacionamentos, associações, ...)
```

Comment:
Maybe it's a bit redundant, from "estática do sistema em blocos de construção" to "estática do sistema em blocos"

Reply:
Changed